projects/sw-offline-google-analytics/README.md

```diff
@@ -15,20 +15,24 @@ Browse sample source code in the [demo directory](https://github.com/GoogleChrom
 
 ### goog.useOfflineGoogleAnalytics
 
-[projects/sw-offline-google-analytics/src/offline-google-analytics-import.js:182-229](https://github.com/GoogleChrome/sw-helpers/blob/213765485d2c3378497e2a80c2b80993a7b09cdf/projects/sw-offline-google-analytics/src/offline-google-analytics-import.js#L182-L229 "Source code on GitHub")
+[projects/sw-offline-google-analytics/src/offline-google-analytics-import.js:45-83](https://github.com/GoogleChrome/sw-helpers/blob/5320c8269f3368939bf792c4bc1a47487ca7963f/projects/sw-offline-google-analytics/src/offline-google-analytics-import.js#L45-L83 "Source code on GitHub")
 
-`goog.useOfflineGoogleAnalytics` is the main entry point to the library
-from within service worker code.
+In order to use the library, call`goog.useOfflineGoogleAnalytics()`.
```

Comment:
Imo it would make sense to switch and be consistent here across projects. Looping in @gauntface and @wibblymat for their take on this just in case I'm a lone voice on this one :)